yolean/pom.xml

```diff
@@ -37,10 +37,6 @@
                 <artifactId>bundle-plugin</artifactId>
                 <extensions>true</extensions>
             </plugin>
-            <plugin>
```

Comment:
Consider keeping the `maven-compiler-plugin` section. It helps IntelliJ to detect correct language level on import.

Reply:
That's why...